base/contents/verify_node

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# fail fast
+set -e
+
+# Bail with a help message if the arguments look wrong
+if [ $# != 2 -o "${1}" == "help" ]; then
+  echo "Usage: ${0} <node-js-binary-file> <SHASUMS256.txt.asc file>"
+  echo ""
+  echo "Verifies the integrity of the given Node.js binary file using the "
+  echo "given SHASUMS256.txt.asc file."
+  echo ""
+  echo "Note: To be valid, the filename of the given Node.js binary must "
+  echo "      match the expected name in the SHASUMS256.txt.asc file."
+  exit 1
+fi
+
+nodejs_file=${1}
+shasums_file=${2}
+
+# Use the absolute path of the KEYS file for safety
+keys_file='/opt/google_appengine/runtime/KEYS'
+
+# Use a temporary file for the keyring file created from the KEYS file
+# to avoid file access conflicts with concurrent invocations of this script
+script_name=`basename ${0}`
+keys_gpg_file=`mktemp /tmp/${script_name}.XXXXXX.keys.gpg`
+
+# Clean up any temporary files created by this script
+function cleanUp {
+  rm -Rf "${keys_gpg_file}"
+}
+
+# The KEYS file needs to be dearmored so that it can be used as a
+# keyring with gpg
+(cat ${keys_file} | gpg --dearmor > ${keys_gpg_file}) \
```

Comment:
I wonder if we should keep a de-armored file in the repo to avoid the need to de-armor at installation time.